resources/views/admin/main.blade.php

```diff
@@ -12,12 +16,14 @@
                     Panel administratora
                 </div>
                 <div class="panel-body">
-                    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-    			    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-				quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-				consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
-				cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
-				proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                    <h2>Witaj w eskulapie.</h2>
+
+                    <p>Ilość zarejstrowanych pracowników w systemie: {{ sizeof($pracownicy) }}</p>
+                    <p>Ilość zarejstrowanych pacjętów w systemie: {{ sizeof($pacjenci) }}</p>
```

Comment:
Wszystko fajnie, ale parę razy powtarza się "pacjętów" i "zarejstorwanych". Poza tym wszystko wydaje się śmigać. Dogłębniej sprawdzę to w swoim zadaniu.